src/App.js

```diff
@@ -1,10 +1,69 @@
-import './App.css';
+import React from "react";
+import { Input, Fab, Box, Typography, Grid } from "@mui/material";
+import AddIcon from "@material-ui/icons/Add";
+import DeleteIcon from "@mui/icons-material/Delete";
+import "@fontsource/roboto/300.css";
+import "./App.css";
 
-function App() {
-  return (
-    <div className="App">
-    </div>
-  );
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      task: "",
+      todo: [],
+    };
+  }
+
+  addTask = (e) => {
+    this.setState({ task: e.target.value });
+  };
+
+  addTodo = (e) => {
+    e.preventDefault();
+    this.setState({ todo: [this.state.task, ...this.state.todo] });
+  };
```

Comment:
I see. You needed `task: ""` to get value from input field, right?
`addTask()` and `addTodo()` are pretty much the same functions, let's omit `addTask()`
Let's do it in a ReactJS way with REFs (REF in react means reference to DOM)
```
constructor() {
...[snip]...
this.todoInput = React.createRef();
}

 render() {
...[snip]...
 <Input
      inputRef={this.todoInput}
      type="text"
      placeholder="Add new ..."
 />
<Fab color="secondary" aria-label="add" onClick={this.addTodo}>
...[snip]...
and just update addTodo() function accordingly.